libyara.NET/packages.config

```diff
@@ -1,5 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="libyara_vs2015_prerelease" version="3.5.1" targetFramework="native" />
-  <package id="yara-vs2015-binary-dependencies" version="0.0.1" targetFramework="native" />
+  <package id="Microsoft.Internal.O365.Security.Native.Vcpkg.Libyara" version="1.0.2" targetFramework="native" />
```

Comment:
public nugets can't take dependencies on internal ones from our feed, since external users won't be able import these and thus won't be able to build. can we build libyara using vcpkg as part of this project?